sadcowpng/sharedspaces/templates/sharedspaces/create_space.html

```diff
@@ -3,8 +3,41 @@
 <head>
     <meta charset="UTF-8">
     <title>Create A Space</title>
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+
+    <style>
+        body {
+            text-align: center;
+        }
+
+        .create_space_form{
+            display: inline-block;
+            width: 800px;
+        }
+
+        .create_space_table{
+            width: 800px;
+        }
+
+        .space_submit_button{
+            margin-top: 10px;
+            width: 400px;
+        }
+    </style>
```

Comment:
Irrelevant for this merge, but I wanted to write it down that in the future we definitely want to consider centralizing styles. That said, that's a later front-end story issue.

Reply:
for now, we did not have a styles.css file so just adding it in the HTML file and moving it when we get all the basic web pages set up.